pyproject.toml

```diff
@@ -4,7 +4,7 @@ build-backend = "scikit_build_core.build"
 
 [project]
 name = "sparse_dot_topn"
-version = "1.0.0"
+version = "1.0.1"
```

Comment:
The version bump should `1.1.0` as this is a new feature and not a patch.

Can you also remove the version change from this commit and add a second commit with `REL: Release v1.1.0` that also documents the change in the changelog

Reply:
Done.